### ISSUE TITLE
NVDA unable to activate links on Apple.com websites using IE9

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#1888]
**Reported by jpgreer on 2011-11-06 12:01**
NVDA is not able to activate links on Apple websites when using Internet Explorer 9 on Windows 7 64 bit. Even bring up the elements list with NVDA+F7  does not activate the links. I have also tested the Apple sites using Firefox and Google Chrome, and they do not have any issues with links.

**Comment by bhavyashah** [comment]
@michaelDCurran Are you still able to successfully reproduce the reported issue? Even if Mick responds in the affirmative, I would still say that IE 9 is more or less dead and I thus suggest closing this ticket as a wontfix regardless. @ehollig

**Comment by michaelDCurran** [comment]
This is no longer an issue in IE 11. Since you can get IE 11 for Windows 7, and it was only specific to one website, now closing.
